lib/path.js

```diff
@@ -115,8 +138,8 @@ win32.resolve = function() {
 
     var result = splitDeviceRe.exec(path),
         device = result[1] || '',
-        isUnc = device && device.charAt(1) !== ':',
-        isAbsolute = win32.isAbsolute(path),
+        isUnc = device && device[1] !== ':',
+        isAbsolute = isUnc || result[2],
```

Comment:
Ah, I'm guessing the answer to my question ^ is so that `win32SplitPath` returns the same data structure as [`posixSplitPath`](https://github.com/joyent/node/blob/v0.12/lib/path.js#L406).